src/main/java/gg/auroramc/collections/config/Config.java

```diff
@@ -46,6 +48,21 @@ public static final class DisplayComponent {
         private String line;
     }
 
+    @Getter
+    public static final class DiscoverMessage {
+        private Boolean enabled;
+        private List<String> message;
+        private Boolean openMenuWhenClicked = false;
+    }
+
+    @Getter
+    public static final class DiscoverSound {
+        private Boolean enabled;
+        private String sound;
+        private Float volume;
+        private Float pitch;
+    }
+
```

Comment:
You don't need to duplicate type. Just use LevelUpMessage. You can rename that class if you want.